app/code/Magento/Ups/Model/Carrier.php

```diff
@@ -77,7 +77,7 @@ class Carrier extends AbstractCarrierOnline implements CarrierInterface
      *
      * @var string
      */
-    protected $_defaultCgiGatewayUrl = 'http://www.ups.com:80/using/services/rave/qcostcgi.cgi';
+    protected $_defaultCgiGatewayUrl = 'https://www.ups.com:80/using/services/rave/qcostcgi.cgi';
```

Comment:
looks like `https://www.ups.com:80/using/services/rave/qcostcgi.cgi` - not calid URL. Probably tit should be like this:
```suggestion
    protected $_defaultCgiGatewayUrl = 'https://www.ups.com/using/services/rave/qcostcgi.cgi';
```